src/utils.cc

```diff
@@ -544,6 +544,8 @@ TGraphAsymmErrors * utils::makeDataGraph(TH1 * dataHist, bool asDensity)
 	double errlow  = (yv-dn);
 	double errhigh = (up-yv);
 
+    if ( int(yv) == 0 ) { errlow=0.0; errhigh = 1.84105; } // ROOT::Math::gamma_quantile_c( (1. - 0.6827)/2.,1,1) ;
```

Comment:
Should this be `int(yv)` or `round(yv)`?
I understand that `RooHistError::instance().getPoissonInterval(int,...)` will also cast instead of rounding, so perhaps we should just pre-process `yv` adequately, possibly using the [StatComm recipe](https://twiki.cern.ch/twiki/bin/view/CMS/PoissonErrorBars) through `SetBinErrorOption(TH1::kPoisson)`.